waspc/data/Generator/templates/server/src/jobs/_job.ts

```diff
@@ -0,0 +1,21 @@
+{{={= =}=}}
+import prisma from '../dbClient.js'
+import type { JSONValue } from '../_types/serialization.js'
+import { createJob, type JobFn } from './{= jobExecutorRelativePath =}'
+{=& jobPerformFnImportStatement =}
+
+const entities = {
+  {=# entities =}
+  {= name =}: prisma.{= prismaIdentifier =},
+  {=/ entities =}
+};
+
+export type {= typeName =}<Input extends object, Output extends JSONValue | void> = JobFn<Input, Output, typeof entities>
```

Comment:
Using `typeof entities` is a great idea, do you think we can use it to simplify how we generate types for Operations and APIs (not for this PR, but generally)?

Reply:
Yep, we could keep the operation and the generic operation type in the same file. 

Then we could do a similar thing like there:
- define `entities` object for the context
- use `typeof entities` for the generic type (we could ditch our whole machinery for transforming entities and tagging them)

Reply:
I'll make an issue for this, thanks!